go.mod

```diff
@@ -1,3 +1,19 @@
 module github.com/submariner-io/cloud-prepare
 
 go 1.13
+
+require (
+	github.com/aws/aws-sdk-go v1.38.9
+	github.com/submariner-io/admiral v0.9.0-m1.0.20210303181719-5bb1de3368b5
+	k8s.io/apimachinery v0.20.5
+	k8s.io/client-go v12.0.0+incompatible
+)
+
+// Pinned to kubernetes-1.17.0
```

Comment:
I think it's because we pin to that in submariner-operator, but may be when being consumed, the pins from the parent project would apply so we don't need to pin here?